draft-patton-cfrg-vdaf.md

```diff
@@ -308,6 +339,19 @@ aggregator 1       aggregator 2             aggregator SHARES
 {: #vdaf-flow title="Execution of a VDAF. The === line represents a broadcast
 channel."}
 
+<!--
+RLB: As above, this diagram makes it look like the aggregators only get the outputs.
+-->
+
+<!--
+RLB: The way you've split these things out, the definition of VDAF is really
+just the definition fo the validation stage of the overall process.  I would
+suggest defining DAF and VDAF as each having all the required parts, just with
+DAF having a null / trivial validation stage.  This is because the relevant
+algorithms (sharding, validation, aggregation, unsharding) all need to be
+compatible with one another, so you need to define them together.
```

Comment:
DAFs and VDAFs are meant to be distinct primitives. They differ in two important ways: VDAFs require the aggregators to interact; and DAFs don't provide any guarantee about the validity of the output. DAFs and VDAFs aren't necessarily "compatible" with one another. It's true that that there may be some overlap between specific constructions, but this isn't necessarily the case.

To be clear, VDAFs are the interesting primitive here. The main reason I defined DAFs is because they are conceptually simpler and provide a nice warm up to VDAFs. Perhaps we should just drop DAFs altogether?